src/sass/edu-benefits.scss

```diff
@@ -25,3 +25,13 @@ select {
 legend {
   @extend h4;
 }
+
+.form-indent {
```

Comment:
I think that's a little harder to read than the design in the wireframes. Could ask Caitlin about it, though.